app/Services/Company/Employee/PersonalDetails/SetPersonalDetails.php

```diff
@@ -28,6 +28,7 @@ public function rules(): array
             'last_name' => 'required|string|max:255',
             'email' => 'required|email:rfc|max:255',
             'phone' => 'nullable|max:255',
+            'timezone' => 'required|integer',
```

Comment:
You should pass the timezone code itself, like 'UTC' or 'Europe/Paris'

Reply:
Excellent idea, this is what i've done.